src/assignment/pkg9/DiceReaderTest.java

```diff
@@ -5,18 +5,63 @@
  */
 package assignment.pkg9;
 
+import java.util.ArrayList;
+import java.util.List;
+import dice.DiceTower;
+
 /**
  *
  * @author Paul Scarrone
  */
-
 // This test passes now and gives an example of how your file should work
 // Once you are done with reading and calculating dice you will need to update
 // this test to pass again
 public class DiceReaderTest {
-  public static void test_reader(){
-	DiceFileReader diceReader = new DiceFileReader("dice.txt");
-	diceReader.read();
-	System.out.println(diceReader.getLines().toString().equals("[]"));
+
+  public static void test_reader() {
+    DiceFileReader diceReader = new DiceFileReader("dice.txt");
+    diceReader.read();
+
+    List<String> lines = new ArrayList<>();
+    lines = diceReader.getLines();
+
+    int trayValue = 0;
+    
+    // break the file down by line
+    for (int i = 0; i < lines.size(); i++) {
+      System.out.println("Tray #" + (i + 1) + " = ");
+      String[] fileLine = lines.get(i).split(" ");
+
+      // seperate the file's line into die
+      for (int ii = 0; ii < fileLine.length; ii++) {
```

Comment:
See, I never knew why everyone always used `i`. I was always taught use whatever you want, never knew of the `i`,`j`, `k` thing.